com.civicrm.hrjobroles/js/src/job-roles/filters/option-group-filter.js

```diff
@@ -0,0 +1,22 @@
+define([
+  'common/lodash',
+  'job-roles/filters/filters'
+], function(_,filters) {
+  'use strict';
+
+  filters.filter('getActiveValues', ['$log', function($log) {
+    $log.debug('Filter: getActiveValues');
+
+    return function(groupData) {
+      var filteredGroupData = [];
+
+      _.each(groupData, function (group) {
+        if (group.is_active === '1') {
+          filteredGroupData[group.id] = group;
```

Comment:
For API call to create new job role, for location, department, level and region, `id` are  no used rather than `value` attribute is used. So in select options in new/edit job role form, we need the option values to contain value as **contents of value property in a optionValue**. So, filter is required to return the key value pair objects. 

`  <select ng-options="k as v.title for (k, v) in jobroles.LocationsData | getActiveValues" ... ></select>`
<img width="1112" alt="screen shot 2017-01-31 at 4 37 03 pm" src="https://cloud.githubusercontent.com/assets/6307362/22461965/8c2466ea-e7d3-11e6-8185-f4c90b463314.png">